packages/framework/src/Form/Admin/Administrator/AdministratorFormType.php

```diff
@@ -99,7 +99,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     private function getFirstPasswordConstraints($scenario)
     {
         $constraints = [
-            new Constraints\Length(['min' => 6, 'minMessage' => 'Password cannot be longer then {{ limit }} characters']),
+            new Constraints\Length(['min' => 6, 'minMessage' => 'Password cannot be longer than {{ limit }} characters']),
```

Comment:
This PR should be rebased on #1478 to avoid conflicts in this change (or resolved after #1478)

Reply:
Yeah :)